elao.app/README.md

```diff
@@ -150,11 +150,20 @@ system:
     # supervisor:
     #     configs:
     #         - file: app.conf
+    #           groups:
+    #               acme:
+    #                   programs:
+    #                       - foo
+    #                       - bar
     #           programs:
-    #               foo-bar:
-    #                   command: zsh -c "bin/console app:foo:bar --no-interaction -vv"
+    #               foo:
```

Comment:
Could you just leave an example program name with an hyphen (`-`) to let people understand this is the standard separator(not underscore)

Reply:
Would this more detailed example work?
(also show's that groups are optional)